packages/react/src/components/TextField/TextField.tsx

```diff
@@ -20,25 +20,49 @@ export type TextFieldProps = {
    *  Use `srLabel` to describe `maxCount` for screen readers.
    */
   characterLimit?: CharacterLimit;
+
   /** The default value for the text field. */
   defaultValue?: string | number;
+
   /** The formatting options for the text field. */
   formatting?: TextFieldFormatting;
+
   /** Specifies whether the value of the text field is valid. */
   isValid?: boolean;
+
   /** The label for the text field. */
   label?: string;
+
   /** Whether the text field is read-only. */
   readOnly?: boolean | ReadOnlyVariant_;
+
   /** The value of the text field. */
   value?: string;
+
   /** Specifies whether the text field is disabled. */
   disabled?: boolean;
+
   /** Specifies whether the text field is required. */
   required?: boolean;
+
```

Comment:
Added line breaks here to make it clear that the comment belongs to the line below and not the one above